src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -175,6 +175,10 @@ paths:
     $ref: 'paths/search_missing_owner.yaml'
   /search/owner:
     $ref: 'paths/search_owner.yaml'
+  /search/package/id:
+    $ref: 'paths/search_package_id.yaml'
```

Comment:
This file is missing, no?

Reply:
That was a leftover of a previous and more aggressive approach to this PR. :sweat_smile: Removed.